### ISSUE TITLE
Bump up yarn version from 1.3.2 to 1.10.0 to get rid of 'deprecated Buffer() constructor' warnings

### ISSUE THREAD

**Issue by Stefie** [polkadot-js/apps#368]
I just did a fresh install and I ran into this issue using Yarn v1.6.0:
https://github.com/yarnpkg/yarn/issues/5477

Older versions of Yarn make use of a deprecated Buffer constructor and therefore throw a security and usability error on install:
`(node:88819) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.`
Also see [Node docs Deprecated APIs](https://github.com/nodejs/node/blob/master/doc/api/deprecations.md#dep0005-buffer-constructor)

It was fixed in Yarn 1.10.0, that's why I'd suggest to change the yarn version in engines in the package.json accordingly.

**Comment by jacogr** [comment]
Yes, been driving me crazy (for quite some time). I was not aware it has been fixed, thank you.
